spec/calculator_spec.rb

```diff
@@ -1 +1,254 @@
-# Your code here
\ No newline at end of file
+require "calculator.rb"
+
+describe Calculator do
+
+  # The add method properly adds numbers,
+  # whether positive or negative, integer or float.
+  describe "add" do
+
```

Comment:
create a common instance for `Calculator` and use it throughout the specs.
`let(:c) { Calculator.new }`